cloudwatch-ecs-event-logs.tf

```diff
@@ -1,4 +1,5 @@
 resource "aws_cloudwatch_log_group" "ecs_events" {
+  count             = var.cloudwatch_logs_create ? 1 : 0
```

Comment:
Do we need to disable the creation of a new CW log group or the aws_cloudwatch_log_resource_policy  only?